schema/cargo.kdl

```diff
@@ -0,0 +1,172 @@
+@kdl:schema "https://github.com/kdl-org/kdl/blob/main/schema/kdl-schema.kdl"
+
+metadata {
+    // TODO: update this link when we're ready to release something.
+    link "https://github.com/kdl-org/kdl/blob/main/schema/cargo.kdl" rel=self
```

Comment:
Likely because I'm super new to this, but what's the value of `rel=...` here?

Reply:
Oh I changed this field to `id`, I just haven’t pushed that change I guess. That’ll be clearer (and not need rel)